### ISSUE TITLE
After re-render chart have duplicated values

### ISSUE THREAD

**Issue by dariuszlacheta** [keen/keen-dataviz.js#171]
After chart is re-render sometimes the same data is doubled, tripled and so on...
![PastedGraphic-4](https://user-images.githubusercontent.com/47319418/55865470-bb6cfe00-5b7e-11e9-989a-b2bb478a104b.png)

**Comment by Tornquist** [comment]
I'm having the same issue. I see that this was closed, but no changes are assigned to this. Was this fixed in a specific version?

**Comment by Tornquist** [comment]
This only happens when I'm caching and re-rendering components. When I build a fresh component and replace the old one no data is duplicated.

**Comment by Tornquist** [comment]
As a note, this happens when I store references to a `KeenDatabiz` object and call render a second time.

When I rebuild new objects and just replace the graph in the container it works without issue and replaces the graph. That's a workable solution because of the full replacement, but it would be nice to be able to call render again.

**Comment by adamkasprowicz** [comment]
Hi @Tornquist 

Let's use `clearOnRender` flag

```javascript
const chart = new KeenDataviz({
  container: '#some_container',
  clearOnRender: true, // clear c3
  transition: {
    duration: 0 // to avoid animation during re-render
  }
});
```

Please let me know if it works for you

**Comment by adamkasprowicz** [comment]
fixed in keen-dataviz@3.8.1
